README.md

```diff
@@ -34,7 +34,9 @@ The `pip-compile` command lets you compile a `requirements.txt` file from
 your dependencies, specified in either `pyproject.toml`, `setup.cfg`,
 `setup.py`, or `requirements.in`.
 
-Run it with `pip-compile` or `python -m piptools compile`. If you use
+Run it with `pip-compile` or `python -m piptools compile` (or
+`pipx run --spec pip-tools pip-compile` if you have
+[configured](#configuration) `pip-tools` appropriately). If you use
```

Comment:
I wonder what you mean by "appropriate configuration"?

Reply:
Yes, and why is `pipx` in this suddenly?

Reply:
Usually it means specifying the Python version or something via `pip-args`. The key thing is since `pipx run` is doing the execution it isn't necessarily going to use the Python version you care about.

Reply:
> Usually it means specifying the Python version or something via `pip-args`.

I don't think that's supported yet? See #1898

Reply:
Ah, true. Misremembered how I solved it for this specific case.

I'll just take the comment out at this point.